gap/prop.gi

```diff
@@ -259,6 +259,10 @@ InstallMethod(IsFunctionalDigraph, "for a digraph by out-neighbours",
 [IsDigraphByOutNeighboursRep],
 D -> ForAll(OutNeighbours(D), x -> Length(x) = 1));
 
+InstallMethod(IsPermutationDigraph, "for a digraph by out-neighbours",
+[IsDigraphByOutNeighboursRep],
+D -> IsFunctionalDigraph(D) and ForAll(InNeighbours(D), x -> Length(x) = 1));
```

Comment:
What you wrote is entirely valid, but I think this way should be slightly more efficient: constructing the in-neighbours of a digraph can be somewhat expensive, but the method for `DigraphSources` uses something called "blists" which I reckon will be a bit faster in the average case.
```suggestion
D -> IsFunctionalDigraph(D) and IsEmpty(DigraphSources(D)));
```